core/domain/wipeout_service.py

```diff
@@ -505,6 +553,14 @@ def verify_user_deleted(
         policies_not_to_verify.append(
             base_models.DELETION_POLICY.DELETE_AT_END)
 
+        # Verify if user profile picture is deleted.
+        user_settings_model = user_models.UserSettingsModel.get_by_id(user_id)
+        username = user_settings_model.username
+        user_roles = user_settings_model.roles
+        if feconf.ROLE_ID_MOBILE_LEARNER not in user_roles:
```

Comment:
All the occurrences where we have a check related to `ROLE_ID_MOBILE_LEARNER` we do it like this only, for reference in function `pre_delete_user` and `delete_user`, that also make sense because if we have a role of mobile learner inside the list of roles we do not want to do anything with the profile picture or deletion. Let me know what do you think.
Thanks

Reply:
Okay, lets keep it as is.